pkg/querier/queryrange/results_cache.go

```diff
@@ -26,12 +26,14 @@ import (
 type ResultsCacheConfig struct {
 	CacheConfig       cache.Config  `yaml:"cache"`
 	MaxCacheFreshness time.Duration `yaml:"max_freshness"`
+	Interval          time.Duration `yaml:"interval"`
```

Comment:
Would be more clear if we name it `SplitInterval` / `split_interval` / `frontend.cache-split-interval` ?

Reply:
Totally agree thanks !